CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Unreleased
 
+## 1.19.0 / 2023-02-27
+
+The module `prometheus/common v0.48.0` introduced a bug when used together with client_golang. If your project uses client_golang and you want to use `prometheus/common v0.48.0` or higher, please update client_golang to v1.19.0.
```

Comment:
Personally I find this frustrating, because I can't figure out what the bug is,
even after reading the full list of commits between 1.18 and 1.19.
Could you give a hint?

Reply:
A few constants ceased to exist (or were made private) in common 0.48.0, client_golang would break when trying to access those constants.

The fix was done here: https://github.com/prometheus/client_golang/pull/1448

Reply:
and now that you mention I totally agree that this text could be clearer, I'm just not sure if users are interested in such explanation "constant values could not be accessed anymore" 🤔

Reply:
Thanks, linking to the fix is enough information to trace it through.
I would say the root of the thing is https://github.com/prometheus/common/pull/576; some constants were deliberately hidden. I would use the word "incompatibility" rather than "bug".

Reply:
We still haven't merged `release-1.19` back to `main`, so I've raised https://github.com/prometheus/client_golang/pull/1454 and we can get it rephrased :)